packages/legacy/core/App/components/texts/Link.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Text, StyleSheet, TextStyle, TextProps } from 'react-native'
+
+import { useTheme } from '../../contexts/theme'
+import { testIdForAccessabilityLabel, testIdWithKey } from '../../utils/testable'
+
+interface Props {
+  linkText: string
+  style?: TextStyle
+  textProps?: TextProps
+  onPress: () => void
+}
+
+const Link: React.FC<Props> = ({ linkText, onPress, style = {}, ...textProps }) => {
```

Comment:
I think by convention we use something like `LinkProps` rather than just `Props`